src/components/cc-tile-metrics/cc-tile-metrics.stories.js

```diff
@@ -0,0 +1,324 @@
+import './cc-tile-metrics.js';
+import './cc-tile-metrics.smart.js';
+import { makeStory, storyWait } from '../../stories/lib/make-story.js';
+import { enhanceStoriesNames } from '../../stories/lib/story-names.js';
+
+export default {
+  title: '🛠 Overview/<cc-tile-metrics>',
+  component: 'cc-tile-metrics',
+};
+
+const conf = {
+  component: 'cc-tile-metrics',
+  // language=CSS
+  css: `cc-tile-metrics {
+    margin-bottom: 1rem;
```

Comment:
don't we always use em unit?

Reply:
I think the whole `css` config is no longer useful in this.